### ISSUE TITLE
Setting log retention for Lambda

### ISSUE THREAD

**Issue by tcabanski** [serverless/serverless#271]
Any way to set this in the JAWS config so it gets pushed to CloudFormation.  If not, any suggestions on how it should be implemented?
